Kernel/ACPI/MultiProcessorParser.cpp

```diff
@@ -226,7 +226,7 @@ PCIInterruptOverrideMetadata::PCIInterruptOverrideMetadata(u8 bus_id, u8 polarit
     , m_polarity(polarity)
     , m_trigger_mode(trigger_mode)
     , m_pci_interrupt_pin(source_irq & 0b11)
-    , m_pci_device_number((source_irq & 0b11111) >> 2)
+    , m_pci_device_number((source_irq >> 2) & 0b11111)
```

Comment:
Is this a bugfix?

Reply:
Yeah, should I give a more detailed description about it in the commit message?

Reply:
Nah, it's ok. I was just a little confused since the PR was called "simplify..." but contained a bugfix :)